raiden/tests/api/test_api.py

```diff
@@ -619,8 +619,8 @@ def test_api_tokens(
     assert_proper_response(response)
     response = response.json()
     expected_response = [
-        {'address': '0x61c808d82a3ac53231750dadc13c777b59310bd9'},
-        {'address': '0xea674fdde714fd979de3edf0f56aa9716b898ec8'},
+        '0x61c808d82a3ac53231750dadc13c777b59310bd9',
+        '0xea674fdde714fd979de3edf0f56aa9716b898ec8',
     ]
     assert all(r in response for r in expected_response)
```

Comment:
This assert would pass if response returned, e.g., a 3rd channel address. Maybe it'd be better to test like:
```
assert set(response) == set(expected_response)
```
?

Reply:
No need since the response itself here is a response of a mock endpoint anyway so I think it was left open-ended on purpose.

Reply:
Hmmm yeah why not. (Disregard my previous comment)